spec/lib/etl/foundation_extraction_spec.rb

```diff
@@ -201,7 +201,7 @@
     end
   end
 
-  describe '#save' do
+  describe '#save', slow_db: true do
```

Comment:
@jsuchal flag pre `DatabaseCleanerHelper` na to aby nastavil sposob cistenia databazy na `:truncation`. Standardne sa totiz pre testy z foldrov: `lib`, `models`, `services`, `mailers`, `helpers`, `decorators` cistia pomocou tranzakcii.

Chcel som pouzit nejaky jednoduchy zamamatovatlny nazov, ale mozem to kludne premenovat